docs/en/smapi_models/ask_smapi_model.rst

```diff
@@ -0,0 +1,15 @@
+SMAPI Models :ask\_smapi\_model package
```

Comment:
Should we keep it 'ASK SMAPI Models' ?

Reply:
Sure, will update it, by amending to same commit since its just a minor change.